WORKSPACE

```diff
@@ -95,7 +95,7 @@ go_x_tools_imports_repositories()
 
 go_googleapis_repositories()
 
-go_istio_api_repositories(False)
+go_istio_api_repositories(True)
```

Comment:
why?

Reply:
https://github.com/istio/api/pull/229 has to be merged first for this to compile.

Reply:
Right, I'll remove this for the final commit.

Reply:
how will this compile even if that is merged? do we have automatic go code gen in istio/api? So far, its manually generated code right? @mandarjog / @ayj ?

Reply:
or @kyessenov

Reply:
from what I know, I'll need to update the SHA in istio_api.bzl once the API PR committed.